sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EpidemiologicalDataCaseSteps.java

```diff
@@ -51,6 +53,26 @@ public EpidemiologicalDataCaseSteps(
       EnvironmentManager environmentManager) {
     this.webDriverHelpers = webDriverHelpers;
 
+    When(
+        "I set Start and End of activity by current date in Activity as Case form for DE version",
+        () -> {
+          webDriverHelpers.fillInWebElement(
+              START_OF_EXPOSURE_INPUT, DATE_FORMATTER_DE.format(LocalDate.now()));
+          webDriverHelpers.fillInWebElement(
+              END_OF_EXPOSURE_INPUT, DATE_FORMATTER_DE.format(LocalDate.now()));
+        });
+    When(
+        "I fill Description field in Activity as Case form",
```

Comment:
*in Case

Reply:
![image](https://user-images.githubusercontent.com/97879405/161941734-6502b8f3-02ea-484b-8050-7ed71ec9ece0.png)

"Activity as Case" is a form case provided by Sormas